contrib/gorm.io/gorm.v1/example_test.go

```diff
@@ -29,11 +37,35 @@ func ExampleOpen() {
 		log.Fatal(err)
 	}
 
-	user := struct {
-		gorm.Model
-		Name string
-	}{}
+	user := User{}
 
 	// All calls through gorm.DB are now traced.
 	db.Where("name = ?", "jinzhu").First(&user)
 }
+
+func Example_context() {
+	// Register augments the provided driver with tracing, enabling it to be loaded by gormtrace.Open.
+	sqltrace.Register("pgx", &stdlib.Driver{}, sqltrace.WithServiceName("my-service"))
+	sqlDb, err := sqltrace.Open("pgx", "postgres://pqgotest:password@localhost/pqgotest?sslmode=disable")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	db, err := gormtrace.Open(postgres.New(postgres.Config{Conn: sqlDb}), &gorm.Config{})
+	if err != nil {
+		log.Fatal(err)
+	}
+
```

Comment:
Please remove the empty lines. See https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines (point 3)